tests/wallet/send.rs

```diff
@@ -106,6 +106,42 @@ fn send_inscribed_sat() {
   );
 }
 
+#[test]
+fn send_inscription_by_sat() {
+  let bitcoin_rpc_server = test_bitcoincore_rpc::spawn();
+
+  let ord_rpc_server =
+    TestServer::spawn_with_server_args(&bitcoin_rpc_server, &["--index-sats"], &[]);
+
+  create_wallet(&bitcoin_rpc_server, &ord_rpc_server);
+
+  bitcoin_rpc_server.mine_blocks(1);
+
+  let (inscription, _) = inscribe(&bitcoin_rpc_server, &ord_rpc_server);
+
+  bitcoin_rpc_server.mine_blocks(1);
+
+  let sat_list = sats(&bitcoin_rpc_server, &ord_rpc_server);
+  let sat = sat_list[0].sat;
+  let address = "bc1qcqgs2pps4u4yedfyl5pysdjjncs8et5utseepv";
+
+  let output = CommandBuilder::new(format!("wallet send --fee-rate 1 {address} {sat}",))
+    .bitcoin_rpc_server(&bitcoin_rpc_server)
+    .ord_rpc_server(&ord_rpc_server)
+    .run_and_deserialize_output::<send::Output>();
+
+  bitcoin_rpc_server.mine_blocks(1);
+
+  let send_txid = output.txid;
+
+  ord_rpc_server.assert_response_regex(
+    format!("/inscription/{inscription}"),
+    format!(
+      ".*<h1>Inscription 0</h1>.*<dt>address</dt>.*<dd class=monospace>{address}</dd>.*<dt>location</dt>.*<dd class=monospace>{send_txid}:0:0</dd>.*",
+    ),
+  );
+}
```

Comment:
Could you add a test, where you only send a sat?

Reply:
> Could you add a test, where you only send a sat?

i have add a `send_uninscribed_sat` test function.